src/routes/create-dictionary.svelte

```diff
@@ -85,6 +85,9 @@
         iso6393: iso6393.trim(),
         glottocode: glottocode.trim(),
       };
+      // Deleting empty fields
```

Comment:
Don't use comments that are redundant. The name of the function says the exact same thing.